qemu/tests/nvdimm_redis.py

```diff
@@ -0,0 +1,44 @@
+import logging
+
+from virttest import error_context
+from virttest import utils_package
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Run nvdimm cases:
+    1) Boot guest with nvdimm device backed by a host file
+    2) Run redis test inside guest
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    vm = env.get_vm(params["main_vm"])
+    session = vm.wait_for_login()
+    error_context.context("Install dependency packages in guest", logging.info)
+    pkgs = params["depends_pkgs"].split()
+    if not utils_package.package_install(pkgs, session):
+        test.cancel("Install dependency packages failed")
+    try:
+        error_context.context("Get redis in guest", logging.info)
+        cmds = []
+        cmds.append(params["get_redis"])
+        cmds.append(params["get_nvml"])
+        cmds.append(params["compile_nvml"])
+        cmds.append(params["compile_redis"])
+        for cmd in cmds:
+            s, o = session.cmd_status_output(cmd, timeout=300)
+            if s:
+                test.error("Failed to run cmd '%s', status: %s, output: %s" %
```

Comment:
I am thinking if it is needed to show the error code `s`, not very necessary, how do you think about remove it ?

Reply:
No big deal to me. Any other comments? I will update together.